go/master/client.go

```diff
@@ -60,13 +60,31 @@ func WithAddr(addr string) func(c *Client) error {
 	}
 }
 
+func retry(f func() error, dur time.Duration, count int) error {
+	err := f()
+	if err != nil {
+		if count > 0 {
+			time.Sleep(dur)
+			return retry(f, dur, count-1)
```

Comment:
I thought should be waiting forever if etcd is not started, this is also what we do in paddle cloud starting up scripts.